packages/compartment-mapper/src/infer-exports.js

```diff
@@ -157,12 +157,27 @@ export const inferExportsAndAliases = (
   tags,
   types,
 ) => {
+  const { name, type, main, module, exports, browser } = descriptor;
+
+  // collect externalAliases from exports and main/module
   assign(
     externalAliases,
     fromEntries(inferExportsEntries(descriptor, tags, types)),
   );
+
+  // expose default module as package root
+  // may be overwritten by browser field
+  // see https://github.com/endojs/endo/issues/1363
```

Comment:
Optional: I try to consistently use capitalization and punctuation to delimit sentences in comments. I also break sentences between lines so that changes to one sentence do not rewrap subsequent sentences.